index.php

```diff
@@ -96,6 +97,35 @@
 		header('Content-Type: application/json');
 		echo json_encode($list, JSON_PRETTY_PRINT);
 
+	} elseif($action === 'detect' && !empty($url)) {
+
+		$format = $params['format']
+			or returnClientError('You must specify a format!');
+
+		foreach(Bridge::listBridges() as $bridgeName) {
+
+			$bridge = Bridge::create($bridgeName);
+
+			if($bridge === false || (!Bridge::isWhitelisted($bridgeName))) {
+				continue;
+			}
+
+			$bridgeParams = $bridge->detectParameters($url);
+
+			if($bridgeParams === null) {
+				continue;
+			}
+
+			$bridgeParams['bridge'] = $bridgeName;
+			$bridgeParams['format'] = $format;
+
+			header('Location: ?action=display&' . http_build_query($bridgeParams));
```

Comment:
The header must include the error code 301 for redirection.

```suggestion
			header('Location: ?action=display&' . http_build_query($bridgeParams), true, 301);
```

See this for reference:

https://github.com/RSS-Bridge/rss-bridge/blob/b29ba5b973694637642659e2bee80e0e4383b58a/index.php#L131-L132